### PR TITLE
Correcting the solution status for SCIP in order to read integer feasible solutions

### PR DESCRIPTION
**Pull request by tomaszG247**
to address issue #451

### CODE REVIEW

File: pulp/pulp.py [hunk 1]
```diff
@@ -59,7 +59,7 @@
 
 Display the status of the solution
 >>> const.LpStatus[status]
-'Optimal'
+'Optimal Solution Found'
```
Comment (pchtsp) [inline pulp/pulp.py:62]: I think this line shouldn't change.

**Review by pchtsp — COMMENTED** [review]
Thanks for the PR! Could you please add a test that failed with the current version and passes with these changes? Feel free to add it for all solvers or only for SCIP in this file: https://github.com/coin-or/pulp/blob/master/pulp/tests/test_pulp.py

if you need to produce feasible but not-necessarily optimal solutions, feel free to use the `create_bin_packing_problem` function together with a timeLimit as in this test: https://github.com/coin-or/pulp/blob/469ef50a037075276e792a3bd5ae43db4fc4fc47/pulp/tests/test_pulp.py#L1148

**Comment by tomaszG247** [comment]
. I added a test as you suggested. Actually it is not passing for both CBC and SCIP. It is clear for me that solution status should be 2.0 (Solution found), but I don't understand what is the expected behaviour for solver status. Is it 'not Solved' or 'Optimal' ? currnetly CBC is retruning optimal and SCIP Not Solved... both are somewhat inadequate IMO. Related to part of the discussion in #476. If you agree I can go with Optimal

File: pulp/pulp.py (hunk 1 above)
Comment (tomaszG247) [reply pulp/pulp.py]: @pchtsp I corrected the pulp.py, must have made an auto-change

**Review by pchtsp — COMMENTED** [review]
Yes, go with Optimal please. This way it will be consistent with other solvers.

Thanks

**Comment by pchtsp** [comment]
Hello again! I've been thinking that it would be nice to add some unit tests that used some sample solution files from SCIP to test these changes.

Also, could you please try to sign the CLA again? I think it broke for a while.

Thanks : )

**Pull request closed without merging**